src/xrCore/LocatorAPI.cpp

```diff
@@ -621,6 +618,7 @@ bool ignore_name(const char* _name)
 
 bool ignore_path(const char* _path)
 {
+    convert_path_separators((char *)_path);
```

Comment:
WTF??? You MUST NEVER do such ugly things like discarding consts! Even if you are going to restore original string there is a chance to get a static string in read-only memory as a parameter. Get rid of this shit, please.